internal/backend/triggers/triggers.go

```diff
@@ -28,20 +31,75 @@ func (m *triggers) Create(ctx context.Context, trigger sdktypes.Trigger) (sdktyp
 
 	trigger = trigger.WithNewID()
 
+	if trigger.ConnectionID() == fixtures.BuiltinSchedulerConnectionID {
+		if schedule, _ := trigger.Data()[sdktypes.ScheduleExpression].ToString(); schedule != "" {
+			scheduleID := newScheduleID(trigger)
+			if err := m.scheduler.Create(ctx, scheduleID, schedule, trigger.ID()); err != nil {
+				return sdktypes.InvalidTriggerID, err
+			}
+			trigger = trigger.WithUpdatedData(sdktypes.ScheduleIDKey, sdktypes.NewStringValue(scheduleID))
+		}
+	}
```

Comment:
```suggestion
	if trigger.ConnectionID() == fixtures.BuiltinSchedulerConnectionID {
		if schedulev := trigger.Data()[sdktypes.ScheduleExpression]; schedule.IsValid() {
            var schedule string
            if err := schedulev.UnwrapInto(&schedule); err != nil {
                return sdktypes.InvalidTriggerID, sdkerrors.NewInvalidRequest(...))
            }
			scheduleID := newScheduleID(trigger)
			if err := m.scheduler.Create(ctx, scheduleID, schedule, trigger.ID()); err != nil {
				return sdktypes.InvalidTriggerID, err
			}
			trigger = trigger.WithUpdatedData(sdktypes.ScheduleIDKey, sdktypes.NewStringValue(scheduleID))
		}
	}
```

Reply:
GetString returns StringValue, from which I need to extract string.
Did you mean `String()` ?